apps/web/app/(app)/environments/[environmentId]/components/EnvironmentLayout.tsx

```diff
@@ -24,12 +25,17 @@ interface EnvironmentLayoutProps {
 }
 
 export const EnvironmentLayout = async ({ environmentId, session, children }: EnvironmentLayoutProps) => {
-  const [environment, organizations, organization] = await Promise.all([
+  const [user, environment, organizations, organization] = await Promise.all([
+    getUser(session.user.id),
     getEnvironment(environmentId),
     getOrganizationsByUserId(session.user.id),
     getOrganizationByEnvironmentId(environmentId),
   ]);
 
+  if (!user) {
+    return <ErrorComponent />;
```

Comment:
Can we change our inconsistent behaviour here. I think we should also here throw an error instead of rendering the error component.